src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -56,6 +56,7 @@
               }
             ],
             "styles": [
+              "node_modules/swagger-ui-dist/swagger-ui.css",
```

Comment:
Adding this here means you're making these styles available on all places and I think this is not necessary.
Have you tried to load it only in the ApiDocsComponent ?

Reply:
Tried adding it in the component, but the css didn't get applied to the api docs page